shared/version/api.go

```diff
@@ -395,6 +395,7 @@ var APIExtensions = []string{
 	"init_preseed_storage_volumes",
 	"metrics_instances_count",
 	"server_instance_type_info",
+	"existing_target_dir_mounted_by_disk_device",
```

Comment:
looking at other disk related api extensions they start with `disk_` so we should keep to the same convention.

Suggest `disk_state_created`